### ISSUE TITLE
colorize registers in dbg.slow

### ISSUE THREAD

**Issue by crowell** [radare/radare2#4714]
colorize registers based on values (pointing to code, etc.) like the stack is done currently, and is done in peda.

**Comment by radare** [comment]
This is already done, but not for dbg.slow

**Comment by XVilka** [comment]
@sivaramaaa see this too

**Comment by sivaramaaa** [comment]
Yup i could give it a try , i too felt the need for it , also **dbg.slow** in genereal itself needs some improvement , for example the register output itself take all the space and u could barely see the instructions !!!

**Comment by radare** [comment]
One option would be to not show the registers that are 0 or only the ones used in context. So we may do 2 passes here. One to get all the regs involved in the pd and then another dr to show only those

**Comment by sivaramaaa** [comment]
Yep that's sounds to be a much better option , i could try to do this !!!

**Comment by radare** [comment]
check the ‘aea’ command

**Comment by radare** [comment]
maybe its easier to modify aea to show the regs like dr= does, with aer= or so than doing it the other way around.

**Comment by ret2libc** [comment]
This issue has been moved from radareorg/radare2 to radareorg/ideas as we are trying to clean our backlog and this issue has probably been created a long while ago. This is an effort to help contributors understand what are the actionable items they can work on, prioritize issues better and help users find active/duplicated issues more easily. If this is not an enhancement/improvement/general idea but a bug, feel free to ask for re-transfer to main repo. Thanks for your understanding and contribution with this issue.
